install.sh

```diff
@@ -60,7 +60,9 @@ is_supported_platform() {
   case "$platform" in
     windows/amd64) found=0 ;;
     darwin/amd64) found=0 ;;
+    darwin/amd64) found=0 ;;
```

Comment:
@AndreasPrang @warrensbox Was this supposed to be `darwin/arm64) found=0 ;;`? I'm having issues installing tgswitch and can't quite track down the core issue yet.

```shell
➜  tmp /usr/local/bin/terragrunt --version
/usr/local/bin/terragrunt: line 7: syntax error near unexpected token `newline'
/usr/local/bin/terragrunt: line 7: `<!DOCTYPE html>'
```

Reply:
I've created a PR to fix this: https://github.com/warrensbox/tgswitch/pull/106